app/controllers/json_web_token.rb

```diff
@@ -0,0 +1,13 @@
+class JsonWebToken < Api::ApiController
+  SECRET_KEY = Rails.application.secrets.secret_key_base.to_s
+
+  def self.encode(payload, exp = 24.hours.from_now)
+    payload[:exp] = exp.to_i
+   
+    JWT.encode(payload, SECRET_KEY)
+  end
+
+  def self.decode(token)
+    JWT.decode(token, SECRET_KEY)[0]
+  end
+end
```

Comment:
This is not a controller, in any case, this should be either a service object or a concern